templates/form_element/datetime-wrapper.html.twig

```diff
@@ -0,0 +1,36 @@
+{#
+/**
+ * @file
+ * Theme override of a datetime form wrapper.
+ *
+ * Available variables:
+ * - content: The form element to be output, usually a datelist, or datetime.
+ * - title: The title of the form element.
+ * - title_attributes: HTML attributes for the title wrapper.
+ * - description: Description text for the form element.
+ * - required: An indicator for whether the associated form element is required.
+ *
+ * @see template_preprocess_datetime_wrapper()
+ */
+#}
+{%
+  set title_classes = [
+    'label',
+    required ? 'js-form-required',
+    required ? 'form-required',
+  ]
+%}
+{% if title %}
+  <h4{{ title_attributes.addClass(title_classes) }}>{{ title }}</h4>
```

Comment:
Are there any accessibility implications for using an h4 here? Strikes me as better just to use a styled div, but might be wrong.

Reply:
Thanks @stephen-cox !
@ekes @tanc thoughts on this?

Reply:
Seems GDS uses h tags for all the titles, but then they are implementing it in their own system too. So there's the header level dance we'd need to work with. Not sure about just styling it either though, as it does have semantic meaning as a heading for the thing. Is there another option?

Reply:
I think we can change this easily enough later with more specific review of general accessibility in multiple cases, or it is easy enough to override in a given theme for a given use case. Thanks for pointing it out though @stephen-cox